specs/authoring/authoring.md

```diff
@@ -74,6 +75,33 @@ commands. For example for `authoring`:
   browser page and you will see the updated version. This process will keep on\
   running until you terminate it with `Ctrl-C`.
 
+# Remote Editing Setup # {#remote-editing}
+
+To quickly edit text in a GitHub repository, you can use the [github.dev](https://docs.github.com/en/codespacesthe-githubdev-web-based-editor)
```

Comment:
The URL is wrong, should be (https://docs.github.com/en/codespaces/the-githubdev-web-based-editor)